docs/cloud/pouta/additional-services.md

```diff
@@ -38,12 +38,25 @@ the CSC Service Desk to coordinate this.
 
 cPouta does not currently offer integrated name service management.
 
-All our floating IPs are mapped to a default hostname, for example
+All our floating IPs are mapped to a default hostname, for example:
 
 ```
 vm0120.kaj.pouta.csc.fi
 ```
 
+To find hostname of a floating IP, you could leverage `host` command:
+```sh
+host -a <floating IP address>
+```
+
+And the result is `vmXXXX.kaj.pouta.csc.fi` listed in output:
+
+```
+... omitted for brevity
```

Comment:
I think you can remove this line. Alternatively, you can just keep the three dots, i.e., without the "omitted for brevity". In my opinion, the three dots already indicate that something else is expected before the subsequent lines, so they are sufficient.

Reply:
fair enough. yes I agree that `omitted for brevity` here is kind of duplicate. thanks.